moped-editor/src/views/projects/projectView/ProjectComponents/index.js

```diff
@@ -164,8 +164,6 @@ export default function MapView({
 
   if (error) console.log(error);
 
-  useZoomToExistingComponents(mapRef, data);
```

Comment:
This is now moved to the map component in `TheMap.js` where we concatenate the project and subproject components and then zoom to them.